playwright/src/test/java/com/microsoft/playwright/TestBrowserContextBasic.java

```diff
@@ -16,32 +16,37 @@
 
 package com.microsoft.playwright;
 
-import org.junit.jupiter.api.Disabled;
+import com.microsoft.playwright.junit.FixtureTest;
+import com.microsoft.playwright.junit.UsePlaywright;
 import org.junit.jupiter.api.Test;
 
 import java.io.OutputStreamWriter;
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.microsoft.playwright.TestOptionsFactories.isFirefox;
+import static com.microsoft.playwright.TestOptionsFactories.isWebKit;
 import static com.microsoft.playwright.Utils.verifyViewport;
 import static java.util.Arrays.asList;
 import static org.junit.jupiter.api.Assertions.*;
 
-public class TestBrowserContextBasic extends TestBase {
+@FixtureTest
+@UsePlaywright(TestOptionsFactories.BasicOptionsFactory.class)
+public class TestBrowserContextBasic {
   @Test
-  void shouldCreateNewContext() {
-    assertEquals(1, browser.contexts().size());
+  void shouldCreateNewContext(Browser browser) {
+    assertEquals(0, browser.contexts().size());
     BrowserContext context = browser.newContext();
-    assertEquals(2, browser.contexts().size());
+    assertEquals(1, browser.contexts().size());
     assertTrue(browser.contexts().indexOf(context) != -1);
     assertEquals(context.browser(), browser);
     context.close();
-    assertEquals(1, browser.contexts().size());
+    assertEquals(0, browser.contexts().size());
```

Comment:
these assertions needed to be updated because the TestBase used to create a context but fixtures don't create one unless you ask for it.

Reply:
We need to be careful to close contexts if (another) test fails, otherwise this might fail as a side effect.